src/Abp/Auditing/AuditingInterceptorRegistrar.cs

```diff
@@ -2,6 +2,7 @@
 using System.Linq;
 using System.Reflection;
 using Abp.Dependency;
+using Abp.Domain.Uow;
```

Comment:
i think it is not needed since `AbpAsyncDeterminationInterceptor` is moved into `Abp.Dependency` namespace?